deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/impl/V1PodHandler.java

```diff
@@ -403,7 +400,7 @@ public int getWeight() {
     public void watch(ApiClient apiClient, String appName, String serviceName, String namespace)
             throws HyscaleException {
         String selector = ResourceSelectorUtil.getServiceSelector(appName, serviceName);
-        PodParent podParent = getPodParent(apiClient, selector, namespace);
+        PodParent podParent = PodParentProcessor.getPodParent(apiClient, selector, true, namespace);
```

Comment:
Why is it static ?

Reply:
Will use HyscaleContextUtil.getSpringBean here as Handler is not a bean?

Reply:
Yeah